### PR TITLE
Detail: Menu toggles

### PR DESCRIPTION
**Pull request by varffs**
- stop main content scroll with open mobile menu
- fix menu-active class when switches between screensizes

### CODE REVIEW

File: css/responsive/mobile.styl [hunk 1]
```diff
@@ -175,6 +175,10 @@ menuBackgrounds()
 
 #menus
   width: 100%
+  &.main-menu-active
+    #main-content
+      .menu-column-content
+        overflow: hidden
```
Comment (mixcas) [inline css/responsive/mobile.styl:181]: wow nice selectors mang

Comment (mixcas) [reply css/responsive/mobile.styl:181]: nah, seriously, this is kinda ugly, no?

Comment (varffs) [reply css/responsive/mobile.styl:181]: whats wrong with it?

Comment (mikeyrayvon) [reply css/responsive/mobile.styl:181]: lol

**Pull request merged**